Lib/test/datetimetester.py

```diff
@@ -1909,9 +1909,6 @@ def test_fromisocalendar_value_errors(self):
             (10000, 1, 1),
             (0, 1, 1),
             (9999999, 1, 1),
-            (2<<32, 1, 1),
-            (2019, 2<<32, 1),
-            (2019, 1, 2<<32),
```

Comment:
I mentioned this in a PR comment:

Re: `datetime.date.fromisocalendar`
Note that this now can throw OverflowError instead of ValueError when
using the C version of datetime. There is precedent for this, e.g.,
`datetime.date.fromordinal` will throw an OverflowError with the C module
but ValueError with the Python.
Also, prior to this PR, the C version `fromisocalendar` would throw a ValueError while the Python version throws TypeError for issues with e.g, missing arguments.

Reply:
Ah, thanks, I didn't make the connection! I'll take another look at that.